app/src/main/java/com/sunasterisk/music_72/utils/MusicUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.sunasterisk.music_72.utils
+
+import com.sunasterisk.music_72.R
+import com.sunasterisk.music_72.data.anotation.GenreName
+
+object MusicUtils {
+
+    fun getGenreImage(@GenreName genre: String) = getGenreImages()[genre]
+
+    private fun getGenreImages() = hashMapOf<String, Int>(
```

Comment:
có rút gọn được đoạn này không nhỉ? 
```suggestion
    private fun getGenreImages() = hashMapOf(
```

Reply:
Dạ được a